react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+
+export default function ToggleSwitch() {
+
+  const [off, on] = useState(false);
+
+  return (
+    <>
+      <label htmlFor="switch" className='toggle'>
```

Comment:
FYI, if you use `<label>` like this, where you wrap it around the element, you don't need the `htmlFor` property, which means you also don't need `id="switch"` on the `input` element.